src/main/scala/extractor/Main.scala

```diff
@@ -12,7 +12,7 @@ object Main {
       case Some(conf) => {
         println(s"Root: ${conf.root}, Multiple projects: ${conf.multidir}")
         val walker = new SingleProjectWalker(conf.root)
-        val results = ImplicitParamsToCSV(walker)
+        val results = walker.run(ExtractImplicits.apply)
```

Comment:
Once you made the ExtractImplicit a function, you do no longer need the apply here